tfhe/src/high_level_api/errors.rs

```diff
@@ -53,6 +53,18 @@ pub enum Type {
     FheUint128,
     #[cfg(feature = "integer")]
     FheUint256,
+    #[cfg(feature = "integer")]
+    FheInt8,
```

Comment:
skipping the "shortints", do we have an issue to remove those from the HL ?

Reply:
I don't think so